test/crio-wipe.bats

```diff
@@ -124,7 +124,7 @@ function start_crio_with_stopped_pod() {
 	CONTAINER_INTERNAL_WIPE=false start_crio_with_stopped_pod
 	stop_crio_no_clean
 
-	run_podman_with_args run --name test -d quay.io/crio/busybox:latest top
+	run_podman_with_args run --name test -d quay.io/sohankunkerkar/fedora-crio-ci:minimal top
```

Comment:
We should put this under quay.io/crio.

Reply:
+1

Reply:
Yeah, I don't have access to push that image under `quay.io/crio`

Reply:
@mrunalp is it okay to use a `fedora-minimal` image instead of a `ubi9-minimal` ?